frontend/app/material-ui-theme.js

```diff
@@ -0,0 +1,115 @@
+import { createMuiTheme } from "@material-ui/core/styles";
```

Comment:
Couldn't we have this file in SCSS like other theming stuff?

Reply:
I would actually maybe prefer using material ui theme, as I do think it could be easier to implement the same themes when changing platform stuff, or?